.github/workflows/ci.yml

```
@@ -6,6 +6,16 @@ on:
 permissions:
   contents: read
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: maven
+    - run: mvn --batch-mode --update-snapshots integration-test
```

```suggestion
    - run: mvn --batch-mode --update-snapshots verify
```

I think we need `verify`, as [I read that](https://stackoverflow.com/questions/26304362/why-failsafe-plugin-requires-both-integration-test-and-verify-goals) this is the stage that actually fails the build if the `integration-test` stage is unsuccessful.

---

Could probably remove `--update-snapshots` too, since we don't have any `SNAPSHOT` dependencies. But that would be a separate PR.

---

Maven/Java can be very bothering sometimes...